docs/installation/ca/Installing_CA_Clone.adoc

```diff
@@ -30,7 +30,7 @@ The command will export the following certificates (including the certificate ch
 
 Note that the existing SSL server certificate will not be exported.
 
-If necessary, third-party certificates (e.g. trust anchors) can be added into the same PKCS #12 file with the following command:
+If necessary, third-party certificates, for example `trust anchors`, can be added into the same PKCS #12 file with the following command:
```

Comment:
Actually, for "trust anchors" I don't think it should be enclosed in backticks since they are indeed English words, not something that you would type as part of the installation process. Maybe we could say "third-party CA certificates" instead?

Reply:
I agree to remove the backticks surrounding words that are names.  I'd like to keep "trust anchors" because we don't know if they are 3rd party certs.  They could be another CA in a different security domain.